### PR TITLE
016 Training card

### PR DESCRIPTION
**Pull request by RuslanTanasiichuk**
create component and services for trainings

### CODE REVIEW

**Review by Romaus — APPROVED** [review]
ok

**Review by yarisleiv — APPROVED** [review]

File: src/app/core/trainings/trainings.service.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+export interface Training {
```
Comment (ViktorRiabinin) [inline src/app/core/trainings/trainings.service.ts:5]: Move it to a separate file in 'types' folder

File: src/app/core/trainings/trainings.service.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  id?: string;
+  country: string;
+  city: string;
+  form: string;
+  technology: string;
+  img: string;
+  data: any;
```
Comment (ViktorRiabinin) [inline src/app/core/trainings/trainings.service.ts:12]: 1- It is 'date'
2 - avoid 'any' type

File: src/app/homepage/training/training.component.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.training_course_container {
+  width: 20rem;
```
Comment (ViktorRiabinin) [inline src/app/homepage/training/training.component.scss:2]: why do you use 'rem' here and 'px' below ?

File: src/app/homepage/homepage.component.ts [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
   selector: 'ia-homepage',
   templateUrl: './homepage.component.html',
   styleUrls: ['./homepage.component.scss']
 })
 export class HomepageComponent implements OnInit {
+  trainings: Training[];
 
-  constructor() { }
-
+  constructor(private trainingsService: DescriptionService) {
```
Comment (ViktorRiabinin) [inline src/app/homepage/homepage.component.ts:14]: Better use the same name for service class and constructor attribute

**Review by ViktorRiabinin — APPROVED** [review]

**Pull request merged**